modules/EnsEMBL/Web/Component/Info/HomePage.pm

```diff
@@ -377,9 +377,7 @@ sub funcgen_text {
         "Go to regulatory feature $sample_data->{'REGULATION_TEXT'}", 'regulation', 'Example regulatory feature'
       ),
       
-      $species eq 'Homo_sapiens' ? '
-        <a class="nodeco _ht _ht_track" href="/info/website/tutorials/encode.html" title="Find out about ENCODE data"><img src="/img/ENCODE_logo.jpg" class="bordered" /><span>ENCODE data in Ensembl</span></a>
-      ' : '',
+      '',
```

Comment:
Could I ask you to find the `%s` in the above `sprintf` that refers to this string, and to remove both this empty string and the `%s` that gets it inserted?

Reply:
I've removed this as requested.